ckanext/security/cache/login.py

```diff
@@ -57,7 +68,7 @@ def needs_lockout(self):
         if self.user is not None and self.count == self.login_max_count:
             log.info("%s locked out by brute force protection" % self.user.name)
             try:
-                notify_lockout(self.user)
+                notify_lockout(self.user, self.login_lock_timeout)
```

Comment:
Passing in the timeout so that the email can tell the account owner how long the account will be locked for.